src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -129,7 +114,39 @@ impl RuntimeHandlerManagerInner {
         }
 
         let config = load_config(spec, options).context("load config")?;
-        self.init_runtime_handler(spec, state, netns, dns, Arc::new(config))
+
+        let mut network_created = false;
+        // set netns to None if we want no network for the VM
+        let netns = if config.runtime.disable_new_netns {
+            None
+        } else {
+            let mut netns_path = None;
+            // get netns path from oci spec
+            if let Some(linux) = &spec.linux {
+                for ns in &linux.namespaces {
+                    if ns.r#type.as_str() != oci::NETWORKNAMESPACE {
+                        continue;
+                    }
+
+                    if !ns.path.is_empty() {
+                        netns_path = Some(ns.path.clone());
+                        break;
+                    }
+                }
+            }
+            // if we get empty netns from oci spec, we need to create netns for the VM
```

Comment:
Ditto. I think using some check function and doing similar operation later when we handle 'netns' is more appropriate. Because I think parameters in sandbox.start() are very important, we should add with restraint as much as possible. Otherwise it would be very easy to introduce the clippy hint of too_many_arugument.

Reply:
How about put `netns `and `network_created` into a struct `network_config`, and passing it to `start`. Because it's hard to check if the network is created in `start()`.

Reply:
I added `SandboxNetworkEnv `struct to store them